JMitnik/haas
# Feature: Unit testing setup for prisma-related API calls

### What has changed?
- initial set up for prisma-related unit tests (See. context.ts in API)
- Wrote 1 example test for the createUser function (Not used anymore but didn't know that before :joy:  ) (See api/src/services/user/UserService.test.ts)
- Moved UserService to services context where it is initiated with a PrismaClient as parameter. This is done in order to be able to initiate the service (and in the future many other services) with a mock prisma client.
- Little cleanup (Removed AddUserView which isn't used anymore)

### How to test
1. Go to api folder and run `yarn test src/services/user/UserService.test.ts`. Make sure the test passes
2. Check database and make sure the entry is not actually created in database
